lib/client/jsx/components/inputs/select_input.jsx

```diff
@@ -6,17 +6,26 @@ const NoneOption = (showNone) => (
   : null
 );
 
-const Option = (v) => (
+const Option = (v,i) => (
```

Comment:
This looks very confusing. I think It makes more sense to do a transformation before passing to Options so that v is always the same type.